ruma-events/src/policy/rule.rs

```diff
@@ -0,0 +1,62 @@
+//! Modules and types for events in the *m.policy.rule* namespace.
+
+use std::fmt::{Display, Formatter, Result as FmtResult};
+
+use serde::{Deserialize, Serialize};
+
+pub mod room;
+pub mod server;
+pub mod user;
+
+/// The payload for policy rule events.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct PolicyRuleEventContent {
+    /// The entity affected by this rule. Glob characters * and ? can be used to match zero or more and one or more characters respectively.
+    entity: String,
+
+    /// The suggested action to take.
+    recommendation: Recommendation,
+
+    /// The human-readable description for the recommendation.
+    reason: String,
+}
+
+/// Rules recommendations
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub enum Recommendation {
```

Comment:
This should have a feature-gated non-exhaustive attribute (see random module in ruma-common).